rustfmt.toml

```diff
@@ -0,0 +1,9 @@
+version = "Two"
```

Comment:
What's wrong with the default format?

Reply:
just be consistent with tikv, not sure why change to version two before

Reply:
I suggest to just use all the default settings of rustfmt. No one is going to sync the settings across repos when there is further change.

Reply:
no need to sync the changes, it's just a initial to make the imports in order. We can change it further on demand regardless of how TiKV does.  Seems version two is a good practice, see https://github.com/rust-lang/rust/blob/master/rustfmt.toml#L2

Reply:
> ..regardless of how TiKV does..

I thought you wanted "be consistent with tikv".

What do you mean "make the imports in order"?

Reply:
`use xxx` is sorted in order